go.mod

```diff
@@ -32,6 +32,7 @@ require (
 	k8s.io/apimachinery v0.18.3
 	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
 	k8s.io/klog/v2 v2.1.0
+	k8s.io/kubernetes v1.14.7
```

Comment:
I don't see a reason for this change.

Does `go mod tidy` keep it?

Reply:
go mot tidy:
-       k8s.io/kubernetes v1.14.7
+       k8s.io/kubernetes v1.14.7 // indirect

Reply:
See 60834e31c20df3c620533dd4cbea80dd2a1395e3

Reply:
You did not use the right Go SDK version! See file `GOLANG_VERSION`. Currently we have v1.14.4.

Reply:
See 1790bc7.